recipes/koopmans-kcp/build.sh

```diff
@@ -0,0 +1,23 @@
+set -ex
+
+export ARCH="x86_64"
+export LIBDIRS="${PREFIX}/lib"
+export BLAS_LIBS="-lblas"
+export LAPACK_LIBS="-llapack"
+export FFT_LIBS="-lfftw3"
+# Override C and Fortran preprocessor
+export CPP="${CC} -E -P"
+export FPP="${FC} -E -P -cpp"
+#export CPPFLAGS="${CPPFLAGS}"
+export CC="mpicc"
+export CFLAGS="${CFLAGS} -L${PREFIX}/lib"
+export FC="mpif90"
+export FFLAGS="${FFLAGS} -fallow-argument-mismatch -L${PREFIX}/lib"
+export LD="mpif90"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+
+./configure \
+    --prefix=${PREFIX}
+
+make kcp
+make install
```

Comment:
Is there a `make check` we  can run here?

Reply:
Good suggestion! I'm implementing it as we speak, and will ping you once it's done and integrated here :slightly_smiling_face:

Reply:
@ocefpaf this is now done :slightly_smiling_face: